### PR TITLE
Kakesiodorobo

### PR DESCRIPTION
**Pull request by d97hah**

### CODE REVIEW

File: languoids/tree/unat1236/iwai1249/md.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
 [core]
-name = Iwaidjan Proper (Unattested)
+name = Iwaidjan_Proper (Unattested)
 level = family
```
Comment (xflr6) [inline languoids/tree/unat1236/iwai1249/md.ini]: regression from https://github.com/clld/glottolog/commit/e62bb4d1943b8e067380a49558beaee120a93969#diff-d77c4a38503185b68b8276ea4c82d706

File: languoids/tree/uncl1493/sere1258/md.ini [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 [core]
-name = Serengeti-Dorobo
+name = Serengeti Dorobo
```
Comment (xflr6) [inline languoids/tree/uncl1493/sere1258/md.ini]: Just to make sure (mybe your tree is outdated), dash to space intended?

**Comment by d97hah** [comment]
argh, good ctach! now fixed

2018-01-17 16:18 GMT+01:00 Sebastian Bank <notifications@github.com>:

**Comment by d97hah** [comment]
yes Serengeti Dorobo with space intended


2018-01-17 16:30 GMT+01:00 Harald Hammarström <harald.hammarstrom@gmail.com>
:

**Comment by xflr6** [comment]
Thanks, will force-push a squashed commit and merge.

**Pull request merged**